certbot-dns-linode/setup.py

```diff
@@ -0,0 +1,69 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+version = '0.23.0.dev0'
+
+# Please update tox.ini when modifying dependency version requirements
+install_requires = [
+    'acme=={0}'.format(version),
+    'certbot=={0}'.format(version),
```

Comment:
It seems like this pattern may have changed. (@bmw: Could you comment on this? It seems like other plugins now express a specific version number here.)

Reply:
#5655 may be a helpful reference.